index.html

```diff
@@ -57,14 +58,292 @@ <h3 class="activity">I am a Full Stack Developer</h2>
             <img src="./images/victory-icon.png" class="icon victory" alt="victory icon">
           </a>
         </li>
+        <li>
+          <a href="#">
+            <img src="./images/twitter-icon.png" class="icon twitter" alt="twitter icon">
+          </a>
+        </li>
         <li>
           <a href="#">
             <img src="./images/m-icon.png" class="icon gmail" alt="gmail icon">
           </a>
         </li>
       </ul>
-      
     </div>
   </section>
+  
+  <section class="mid-section">
+    <div class="my-recent-work">
+      <p>My Recent Works</p>
+    </div>
+    <div class="mid-border">
+      <img src="./images/border.png" alt="border">
+    </div>
+    <div class="feature-image">
+      <div class="img1">
+        <img src="./images/placeholder-img.png" alt="img1">
+      </div>
+      <div class="title1">
+        <p class="multi-post-stories">Multi-Post Stories</p>
+      </div>
+      <div class="sup-txt1">
+        <p class="description2">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <ul class="wrapper1">
+        <li class="box1 css">css</li>
+        <li class="box1 html">html</li>
+        <li class="box1 bootstrap">bootstrap</li>
+        <li class="box1 ruby">Ruby</li>
+      </ul>
+      <div class="see-project-btn">
+        <button class="btn1">See Project</button>
+      </div>
+    </div>
+  </section>
+
+<!-- Card Starts here-->
+<!-- Card 1-->
+  <section class="cards-details">
+      <div class="card-container">
+        <div class="title2">
+          <h1>Professional Art Printing Data</h1>
+        </div>
+        <div class="sup-txt2">
+          <p class="description3">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+            standard
+          </p>
+        </div>
+        <ul class="wrapper1">
+          <li class="box2 html">html</li>
+          <li class="box2 bootstrap">bootstrap</li>
+          <li class="box2 ruby">Ruby</li>
+        </ul>
+      </div>
+      <div class="card-project-btn">
+          <button class="btn2">See Project</button>
```

Comment:
- [ OPTIONAL ] I noticed that you're missing the button interaction styles. I will suggest that even though this is an optional comment, you add them to help enable user interactivity in your project :+1: and also so you can practice the skills you've learned so far :100: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/219853637-2eba02ac-8c69-431f-a162-bae0a002daa5.png)